frontend/src/movie-gallary/movie-gallary-root/movie-gallary-root.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import MovieTileItem from '../movie-gallary-item/movie-gallary-item';
+
+import './movie-gallary-root.css';
+
+export default class MovieTileGallary extends React.Component {
+    render() {
+        if(this.props.movies === null || this.props.movies.length === 0) {
```

Comment:
Why the strict comparison to `null`? What will happen if `movies` aren't passed (undefined)?